CONTRIBUTING.md

```diff
@@ -1,51 +1,84 @@
-# Contributing to linkerd #
+# Contributing to linkerd
 
-:balloon: Thank you for using the linkerd project!
+:balloon: Thanks for your help improving the project!
 
-## Getting Help ##
+## Getting Help
 
-If you have a question about linkerd or have encountered problems
-using linkerd, you may [file an issue][issue] or join us on
-[Slack][slack].
+If you have a question about linkerd or have encountered problems using it, you may [file
+an issue][issue] or join us on [Slack][slack].
 
-## Submitting Changes ##
+## Submitting a Pull Request
 
-If you'd like to make a change to linkerd, you should do the
-following:
+Do you have an improvement?
 
 1. Submit an [issue][issue] describing your proposed change.
-2. We will respond to your issue promptly.
-3. If your proposed change is accepted, and you haven't already done
-so, sign [Buoyant's Contributor License Agreement][cla].  Before we
-can accept your patches, our lawyers would like to be assured that:
-    - The code you're contributing is yours, and you have the right to
-    license it.
-    - You're granting us a license to distribute said code under the
-    terms of this agreement.
-4. Fork the linkerd repo, develop and test your code
-changes. See the project's [README](README.md) for further information
-about working in this repository.
-5. Submit a pull request against linkerd's `master` branch.
-6. Your branch may be merged once all configured checks pass,
-including:
+2. We will try to respond to your issue promptly.
+3. If your proposed change is accepted, and you haven't already done so, sign our [Contributor License Agreement][cla].
+4. Fork this repo, develop and test your code changes. See the project's [README](README.md) for further information about working in this repository.
+5. Submit a pull request against this repo's `master` branch.
+6. Your branch may be merged once all configured checks pass, including:
     - 2 code review approvals, at least 1 of which is from a buoyant member
```

Comment:
Rather than buoyant, it might be better to change this requirement to a "linkerd organization member" or something similar.